test/library/packages/UnitTest/GetTest/Primitive_Get_Test.chpl

```diff
@@ -0,0 +1,27 @@
+use UnitTest;
+//verifies that generic single argument functions aren't counted
+
+/* A normal function which takes a generic single argument*/
+proc dummy_test(test) throws {
+    //This is a dummy test
+    writeln("a");
+}
+
+/* A test function which takes Test Object as argument*/
+proc test_square(test: borrowed Test) throws {
+    // This is a test function
+    writeln("b");
+}
+
+/* A test function which takes Test Object as argument*/
+proc test_squared(test: borrowed Test) throws {
+    // This is a test function
+    writeln("c");
+}
+var test: owned Test;
```

Comment:
Doesn't this mean that test is `nil`?  Or am I behind on the nilability changes?

Reply:
In these primitives, we just need to check the type. So a variable that can hold these will work.